include/lsst/ip/diffim/DipoleAlgorithms.h

```diff
@@ -299,10 +303,10 @@ class PsfDipoleFlux : public DipoleFluxAlgorithm {
     {
         meas::base::CentroidResultKey::addFields(schema, name+"_pos_centroid", "psf fitted center of positive lobe", meas::base::SIGMA_ONLY);
         meas::base::CentroidResultKey::addFields(schema, name+"_neg_centroid", "psf fitted center of negative lobe", meas::base::SIGMA_ONLY);
-        meas::base::CentroidResultKey::addFields(schema, name+"_centroid", "average of negative and positive lobe positions", meas::base::SIGMA_ONLY);
+        meas::base::CentroidResultKey::addFields(schema, name, "average of negative and positive lobe positions", meas::base::SIGMA_ONLY);
         _posCentroid = meas::base::CentroidResultKey(schema[name+"_pos_centroid"]);
         _negCentroid = meas::base::CentroidResultKey(schema[name+"_neg_centroid"]);
-        _avgCentroid = meas::base::CentroidResultKey(schema[name+"_centroid"]);
```

Comment:
Removing the suffixes seem like it could break other codes, or user scripts. For something that's going to be removed soon, is it necessary? At the very least, it should be on its own separate commit.

Reply:
Unfortunately, if I don't change it some existing tests (that use DipoleMeasurementTask, which are only used in tests now and one of the things I'd like to deprecate in the future) will fail because they set `ip_diffim_PsfDipoleFlux` as the centroid slot, and the centroid slot requires that "name_x"/"name_y" be the centroid fields, not "name_centroid_x".

Nothing in the stack has used these values for a long time, so I think we're ok.

Reply:
Alright, then!